polaris-react/src/components/Filters/Filters.scss

```
@@ -1,295 +1,261 @@
 @import '../../styles/common';
 
-$list-filters-header-height: $top-bar-height;
-$list-filters-footer-height: 70px;
+$border-width: 1px;
+$border-radius: 32px;
```

```suggestion
$border-width: var(--p-border-width-1);
$border-radius: 32px;
```
Do we have guidance around when/when not to add tokens? cc @alex-page @lgriffee @heyjoethomas

The current border-radius tokens use a 1rem/4px scale from `--p-border-radius-1` to 20px for `--p-border-radius-5`, but then jumps to 30px for `--p-border-radius-6`. Is 30px the max intentionally, or could @mrcthms add support for 32px border-radius in the shape tokens?

---

I can use `--p-border-radius-6` here, no need to add a custom token :+1:

---

Not sure why we use 30px.

For the question around adding tokens, it really is about does this decision need to scale across the admin. We just added a number of color tokens for Shopify Magic because that platform will be showing up across the admin and leveraged by a lot of folks in the future. So it was worth the time to generate a new color palette and create new tokens. 

I would say the bar for adding tokens should generally be pretty high since they are super foundational pieces.

---

I don’t think `--p-border-radius-6` having a values of `30px` and being off the `4px` scale is intentional. Some of our token groups haven’t been really polished or finalized (some still have old values in them) and I bet that’s one of the values that we haven’t addressed yet.

From a quick look at the admin there are:

| Token | # Instances | 
| ----- | ------- | 
| `--p-border-radius-05` | 18 instances |
| `--p-border-radius-1` | 478 instances |
| `--p-border-radius-2` | 463 instances|
| `--p-border-radius-3` | 39 instances |
| `--p-border-radius-4` | 6 instances|
| `--p-border-radius-5` | 14 instances|
| `--p-border-radius-6` | 12 instances |
| `border-radius: 32px` | 2 instances |

Based on this, perhaps we should remove the `--p-border-radius-6` token altogether and create a new `--p-border-radius-8` token with a value of `32px` so that it follows the same scale as the rest of our `border-radius` tokens and migrate any instances of `--p-border-radius-6` to `--p-border-radius-8`. 

If we decide to do that, I can add a `--p-border-radius-8` token with a value of `32px` and remove/migrate of all `--p-border-radius-6` tokens in `v11`. 

Personally I'd love to audit our tokens to get a better sense of which values are more "legacy" than others and need to be addressed (so things like this don't slip through the cracks or cause confusion). And on top of that actually audit what values are being used in the admin and support that / determine how they fit with our design direction.